postcss.config.js

```diff
@@ -0,0 +1,54 @@
+const autoprefixer = require('autoprefixer')
+const cssnano = require('cssnano')
+const oldie = require('oldie')
+const pseudoclasses = require('postcss-pseudo-classes')
+
+/**
+ * PostCSS config
+ *
+ * @type {import('postcss-load-config').ConfigFn}
+ */
+module.exports = ({ env, file }) => {
+  const plugins = [
+    autoprefixer({ env })
+  ]
+
+  // Minify CSS
+  if (file.basename.endsWith('.min.css')) {
```

Comment:
Neat!

Reply:
Due to tooling file extension differences (source vs target) this is now `name.endsWith('.min')`